sdk/python/Makefile

```diff
@@ -18,23 +18,24 @@ TEST_ALL_DEPS = build
 
 ensure::
 	$(PYTHON) -m venv venv
-	. venv/*/activate && python -m pip install -r requirements.txt
+	. venv/*/activate && pip install wheel
+	. venv/*/activate && pip install -r requirements.txt
 	mkdir -p $(PYENVSRC)
 
-build_package::
+build_package:: ensure
```

Comment:
That one is interesting - I'm wondering why our prior Makefile art does not specify dependencies at all. Given that `ensure` does not cache, this makes it run over and over. But package downloads cache. So it comes to about 3s on my machine. So I think this is a good change to take if it makes dev workflow less confusing to start with, but I wonder what others think.